setup.py

```diff
@@ -52,12 +52,20 @@ def run(self):
 requires = []
 for dep in info.get('depends', []):
     if not re.match(r'(ir|res|webdav)(\W|$)', dep):
-        requires.append(
-            'trytond_%s >= %s.%s, < %s.%s' % (
-                dep, major_version, minor_version,
-                major_version, minor_version + 1
+        if dep == 'product_notebook':
```

Comment:
The prefix of product_notebook is openlabs, if I run ```python setup.py install```, it will look for `trytond_product_notebook` which is not available, So I changed it, if dependency is `product_notebook`, then look for `openlabs_product_notebook`, not `trytond_product_notebook`